packages/protocol/contracts/signal/SignalService.sol

```diff
@@ -144,7 +162,7 @@ contract SignalService is EssentialContract, ISignalService {
     /// @return The unique storage slot of the signal which is
     /// created by encoding the sender address with the signal (message).
     function getSignalSlot(
-        uint256 chainId,
+        uint64 chainId,
```

Comment:
@cyberhorsey be careful with this one (if you need to implement similar on relayer side.. if you rely on this call it's ok), because it modifies the hash ! (uint256 vs uint64)

Reply:
Should we simply change the code here to use uint256?

Reply:
ty, I have already covered it in the relayer PR!